week3/src/components/LevelButton.jsx

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable react/prop-types */
+import styled from "styled-components";
+
+function LevelButton({ setCards, setDifficulty, getMixedCardList }) {
+  const handleEasy = () => {
+    setCards(getMixedCardList(5));
+    setDifficulty("easy");
+  };
+
+  const handleNormal = () => {
+    setCards(getMixedCardList(7));
+    setDifficulty("normal");
+  };
+
+  const handleHard = () => {
+    setCards(getMixedCardList(9));
+    setDifficulty("hard");
+  };
```

Comment:
p5: 오오 비슷한 동작을 하는 함수가 3개 있군요~ 공통적으로 동작하는 기능을 하나의 함수로 관리해서, 카드 개수와 난이도를 prop으로 주는 방법도 소개해드릴게요~

```suggestion
const handleChangeLevel = (difficulty,count) => {
      setCards(getMixedCardList(count));
      setDifficulty(difficulty);
}
```

Reply:
그러면 훨씬 코드 재사용성이 높아지겠네요 !!